src/cmds/shell.rs

```diff
@@ -1,19 +1,13 @@
 use std::error::Error;
 
-use crate::filesystem;
-
 pub fn main(shell: &str) -> Result<(), Box<dyn Error>> {
-    let file = match shell {
-        "zsh" => "navi.plugin.zsh",
-        "fish" => "navi.plugin.fish",
-        _ => "navi.plugin.bash",
+    let content = match shell {
+        "zsh" => include_str!("../../shell/navi.plugin.zsh"),
+        "fish" => include_str!("../../shell/navi.plugin.fish"),
+        _ => include_str!("../../shell/navi.plugin.bash"),
```

Comment:
Isn't there a way to create a filesystem path that's platform specific?
Windows would prefer `\` rather than `/` (even though both work).

Reply:
This is at compile time. It's the same as if I copy pasted the contents of file to the source code. I think rust handles the / vs \ thing

Reply:
Ohhh, didn't know that `include_str` macro actually reads the file in _compile time_!

So, now that we're passed that, why was this done?
I'm assuming that's because you didn't want to add these files as external resources so it'll be easier to bundle with additional packaging methods?

If that's the case, then you can write that file to a similar place like the cheats and then print the path to that file.

But, it's a super simple change from the users perspective so I wouldn't bother going to all that trouble.
Plus, it's easier to change the bound key if you want to - instead of reading the output of the command and then regexing & replacing the key, you just need to regex & replace the key.